### ISSUE TITLE
Attendance Details Workflow does not launch

### ISSUE THREAD

**Issue by markewampler** [SparkDevNetwork/Rock#4198]
### Description
We have setup attendance to be taken by group leaders using the GroupLeader Toolbox built into Public Facing Website.  The leader logs in and records attendance and it works correctly as expected.  However we have setup a workflow to launch using the Group Attendance Detail Block settings.  The workflow never seems to launch.

![](https://user-images.githubusercontent.com/33845027/80026936-8d680c80-84a8-11ea-86a8-d511b79291cb.png)

### Steps to Reproduce
1. Enable a group for attendance by adjusting the grouptype
1. make a leader role who can take attendance
1. setup a workflow to launch by editing the block settings and choosing the workflow
1. take attendance as a leader and check workflow logs and see no workflow launched

**Expected behavior:**

A workflow should launch when attendance is saved.

**Actual behavior:**

No workflow is launched when attendance is saved.

### Versions
* **Rock Version:** [10.1]
* **Client Culture Setting:** [en-us]

┆Attachments: <a href="https://app.asana.com/app/asana/-/get_asset?asset_id=1176530967576205">image.png</a>

**Comment by nairdo** [comment]
@markewampler  We're unable to reproduce this issue, but would you be willing to attach an exported copy of your workflow for us to test?

**Comment by markewampler** [comment]
@nairdo Here you go.
[(Emailer) Regen Group Attendance Notes_202005181936.json.zip](https://github.com/SparkDevNetwork/Rock/files/4647578/Emailer.Regen.Group.Attendance.Notes_202005181936.json.zip)

**Comment by nairdo** [comment]
We spoke to Mark via Chat. Since this is not happening on a clean environment he's investigating further to see if there is a localized problem.

**Comment by markewampler** [comment]
Okay.  Update on this.  We were able to reproduce this on the demo site today.  We are getting an exception this time though.

The attendance does record in the tables it seems, but something is not happening correctly.

![Screen Shot 2020-07-07 at 2 12 06 PM](https://user-images.githubusercontent.com/33845027/86832157-6e1b5d00-c05d-11ea-8724-4b8a7b28e2f0.png)

**Comment by nairdo** [comment]
Thanks @markewampler   Using your setup (copied everything locally) I was able to track down the root cause of that exception.   The code is expecting that the group would have a "Schedule" however in your example, it's null so this is throwing the exception:

https://github.com/SparkDevNetwork/Rock/blob/f8cb4396d82fe6993cd900ea2f22542b094c5978/RockWeb/Blocks/Groups/GroupAttendanceDetail.ascx.cs#L1094

**Comment by markewampler** [comment]
Would that be causing it to freeze and not launch a workflow as that command is given after the schedule exception then?

**Comment by nairdo** [comment]
@markewampler yes, because of that exception it never processes the workflow.  We're going to get this fixed in v11.1:
```CSharp
new WorkflowService( rockContext ).Process( workflow, _group, out workflowErrors );
```

As a workaround, it should work correctly if you just set up a schedule for that group.  _Our fix will be to not try to set the Schedule attribute for the workflow if there is no group schedule._
